sources/String.move

```diff
@@ -0,0 +1,77 @@
+module SFC::String {
```

Comment:
Should we use another module name other than `String`? Maybe in the future, Move has the native string implementation.

Reply:
maybe use `Strings`, `StringUtil`, `StringHelper` or something else?

Reply:
StringUtil